nni/algorithms/compression/v2/pytorch/utils/pruning.py

```diff
@@ -21,6 +21,21 @@ def config_list_canonical(model: Module, config_list: List[Dict]) -> List[Dict]:
             else:
                 config['sparsity_per_layer'] = config.pop('sparsity')
 
+    for config in config_list:
+        if 'op_partial_names' in config:
+            op_names = []
+            for partial_name in config['op_partial_names']:
+                for name, _ in model.named_modules():
+                    if partial_name in name:
+                        op_names.append(name)
+            if 'op_names' in config:
+                config.pop('op_partial_names')
+                config['op_names'].extend(op_names)
+                config['op_names'] = list(set(config['op_names']))
+            else:
+                config['op_names'] = config.pop('op_partial_names')
```

Comment:
why `config['op_names'] = config.pop('op_partial_names')`? seems this `config['op_names']` is never used.

Reply:
some legacy logic parts have crushed and I optimized this jut now : (